--- libplist-2.0.0/m4/ac_python_devel.m4~	2015-01-31 20:18:32.000000000 +0100
+++ libplist-2.0.0/m4/ac_python_devel.m4	2019-11-02 21:47:11.479855891 +0100
@@ -169,10 +169,10 @@
 	# Check for Python library path
 	#
 	AC_MSG_CHECKING([for Python library path])
-	if type $PYTHON-config; then
-		PYTHON_LDFLAGS=`$PYTHON-config --ldflags`
-	fi
 	if test -z "$PYTHON_LDFLAGS"; then
+		if type $PYTHON-config; then
+			PYTHON_LDFLAGS=`$PYTHON-config --ldflags`
+		fi
 		# (makes two attempts to ensure we've got a version number
 		# from the interpreter)
 		py_version=`$PYTHON -c "from distutils.sysconfig import *; \
